nix/t1/vcs.nix

```diff
@@ -29,12 +29,12 @@ let
         -sverilog \
         -full64 \
         -timescale=1ns/1ps \
-        -P $VERDI_HOME/share/PLI/VCS/LINUX64/novas.tab $VERDI_HOME/share/PLI/VCS/LINUX64/pli.a \
         ${lib.optionalString vcs-dpi-lib.enable-trace ''
+          +define+T1_ENABLE_TRACE \
           -debug_access+pp+dmptf+thread \
```

Comment:
why doing this?

Reply:
To enable fsdb dumping for VCS, debug_access is sufficent, no need to specify PLI lib manually.

Enable tracing or debug_access may potentially prevent optimization thus make simulation a bit slower. (In theory, I havn't measure it). Therefore I'd like to gate wave dumping code completely.